src/frontend/youtube_clone/src/app/main-page/main-page.component.spec.ts

```diff
@@ -0,0 +1,21 @@
+import { ComponentFixture, TestBed } from '@angular/core/testing';
```

Comment:
That file still exist

Reply:
i think i submitted to the next pull request